### PR TITLE
docs: change current and next version for the docs

### PR DESCRIPTION
**Pull request by IamLizu**
As we have [Express 5 stable released](https://github.com/expressjs/express/releases/tag/v5.0.0),

- changed the next version to _5.0.0_
- changed current version to _4.20.0_ as of current latest

### CODE REVIEW

**Comment by IamLizu** [comment]
Perhaps we have a lot to do in terms of website doc change as _5.0.0_ is not in beta anymore.

We should,

- [ ] Prepare a list of files that has to be changed in accordance with the _5.0.0_ release.
- [ ] Create separate issues from the list so that people from the community can take ownership and contribute.

cc: @expressjs/docs-wg

**Review by UlisesGascon — APPROVED** [review]

File: _data/express.yml [hunk 1]
```diff
@@ -1,2 +1,2 @@
-current_version: "4.19.2"
-next_version: "5.0.0-beta.1"
\ No newline at end of file
+current_version: "4.20.0"
```
Comment (bjohansebas) [inline _data/express.yml]: We should promote version 5 instead of version 4
```suggestion
current_version: "5.0.0"
```

Comment (IamLizu) [reply _data/express.yml]: We could, but our current version is not 5 yet. We are still calling it next until we polish our documentation or other things first. 

In short, `4.20.0` will be _lastest_ and `5.0.0` will be _next_ for a while.

Comment (bjohansebas) [reply _data/express.yml]: Sure, I have more context now, so it seems good to me.

**Comment by bjohansebas** [comment]
@IamLizu, there’s already an issue that provides quite a good overview of all the changes that need to be made to the API and the documentation. Maybe it’s missing some minor changes introduced after beta 3, but I think it’s pretty good for that. see #1479

File: _data/express.yml (hunk 1 above)
Comment (bjohansebas) [inline _data/express.yml]:
```suggestion
current_version: "4.21.0"
```

**Comment by IamLizu** [comment]
Hi @crandmck 

I think its good to go now, requesting your feedback.

**Review by bjohansebas — APPROVED** [review]

**Review by carlosstenzel — APPROVED** [review]

**Review by crandmck — APPROVED** [review]

**Pull request merged**